documentation/Windows Event Log (EVT) format.asciidoc

```diff
@@ -291,8 +298,16 @@ path specification of the event message file.
 ....
 
 [NOTE]
-This value can contain multiple filenames separated by a semi colon (;)
-character and that the name of the event message files is case insensitive.
+If set, this value must container one or more filenames separated by a semicolon (;)
+character. The names of the files are case insensitive. Values that are a directory
+or an invalid file (see below) are ignored. Filenames may contain case-insensitive
+environment variables that must be expanded.
+
+[NOTE]
+Each program that displays event message strings is responsible for reading the
+registry value, handling any environment variables within the value, and calling
+LoadLibrary on each file. Programs other than the Event Viewer may treat
```

Comment:
"calling LoadLibrary on each file", this is not a requirement. evtexport does not but can still read the message strings.

Reply:
Fair point. I can rephrase this.